src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  padding-right: 243px;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    padding: 0 50px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .header-logo {
+    padding-right: 419px;
+  }
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
It's a good practice to be consistent with your vertical margins. For instance, add only top or only bottom margin, but not both. This is due to the fact that vertical margins can be collapsed in some cases.